src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/LiveMetricsTests.cs

```diff
@@ -286,89 +220,6 @@ public async Task TestSystemDiagnosticsMetrics()
                 });
         }
 
-        [Theory]
-        [InlineData(MetricProviderType.All, true)]
-        [InlineData(MetricProviderType.Meter, true)]
-        [InlineData(MetricProviderType.EventCounter, false)]
-        public async Task TestSystemDiagnosticsMetrics_MetricProviderType(MetricProviderType metricType, bool expectResults)
```

Comment:
Are these tests subsumed by other tests?

Reply:
These were just to test the MeterType mechanism (that we get results for Meter/All but not EventCounter) - if we aren't exposing that as configuration anymore I'm assuming we don't need them anymore